samples/snippets/pom.xml

```diff
@@ -41,9 +41,14 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-document-ai</artifactId>
+      <!-- force version to pick up 0.1.1 -->
+      <version>0.1.1</version>
```

Comment:
```suggestion
```

Reply:
it automatically goes back to 0.1.0. I cannot figure out how to force update on bom

Reply:
Alright, at least delete the comment. `<!-- force version to pick up 0.1.1 -->`